cni/network/invoker_azure.go

```diff
@@ -77,16 +75,15 @@ func (invoker *AzureIPAMInvoker) Add(nwCfg *cni.NetworkConfig, _ *cniSkel.CmdArg
 			nwCfg6.Ipam.Subnet = invoker.nwInfo.Subnets[1].Prefix.String()
 		}
 
-		resultV6, err = invoker.plugin.DelegateAdd(nwCfg6.Ipam.Type, &nwCfg6)
+		ret.ipv6Result, err = invoker.plugin.DelegateAdd(nwCfg6.Ipam.Type, &nwCfg6)
 		if err != nil {
 			err = invoker.plugin.Errorf("Failed to allocate v6 pool: %v", err)
 		}
 	}
 
-	sub := &result.IPs[0].Address
-	*subnetPrefix = *sub
+	*ret.subnetPrefix = ret.ipv4Result.IPs[0].Address
```

Comment:
shouldn't this be ret.subnetPrefix = ret.ipv4Result.IPs[0].Address?

Reply:
nope this is correct since subnetPrefix is *